starter-code/iteration1/src/app/iteration2/iteration2.component.html

```diff
@@ -0,0 +1,23 @@
+<hr><p class="header">Iteration 2</p><hr>
+
+<div class="container">
+  <div class="row">
+    <div class="col-xs-12 header">
+      <h1 class="centered"> Learn how to code with Ironhack </h1>
+        <img class="img-rounded"src="./assets/photos/ironhack.jpg" alt="">
```

Comment:
This here could be considered as an article that has title, image and description. Since we are practicing making dynamic page, we don't want to hard code anything. You've already applied similar things in this project, I know you know how to do it - I just want you to remember this - the 3rd module is all about dynamic SPA.